packages/desktop/src/renderer/components/widgets/channels/ChannelInput/ChannelInput.tsx

```diff
@@ -472,7 +490,7 @@ export const ChannelInputComponent: React.FC<ChannelInputProps> = ({
                       setFocused(true)
                     }
                   }}
-                  disabled={inputState !== INPUT_STATE.AVAILABLE}
+                  disabled={disableInput}
```

Comment:
Instead of adding logic in the component, I think you should only pass INPUT_STATE down to it